ckanext/opendatani/helpers.py

```diff
@@ -140,21 +141,38 @@ def is_admin(user, org):
         [(i.get('capacity') == 'admin' or i.get('sysadmin'))
          and i.get('name') == org for i in user_orgs])
 
-    if not result:
-        toolkit.abort(403, _('You are not authorized to access this \
-                        report or the organization does not exist.'))
-    return
+    return True if result else False
+
+
+def verify_datasets_exist(org):
+    """
+    Returns the number of datasets (including private) for a given organization
+    :param org: organization name
+    :type org: string
+    :returns: dataset count
+    :rtype: integer
+    """
+
+    return toolkit.get_action('package_search')({}, {
+        'q': 'organization:{0}'.format(org),
+        'include_private': True}).get('count')
 
 
 def prepare_reports(org):
-    """Creates json file and stores it under CKAN's storage path.
-    :return: a string containing the file_name of the created archive
-    :rtype: string
+    """
+    Creates CSV and JSON files, and stores them under CKAN's storage path.
+    :param org: organization
+    :type org: string
+    :return: a list containing the file_names of the created archives
+    :rtype: list
     """
 
     resource = toolkit.get_action(
         'report_resources_by_organization')({}, {'org_name': org})
 
+    if not resource:
+        return [org, org]
```

Comment:
`return [org, org]` Can you explain this? And comment in the code as well? Why not `None` or `[]`

Reply:
It was due to how I had set some variables in my template. I fixed it by moving them within my if statement. This code will be removed.